youtube_dl/extractor/qqmusic.py

```diff
@@ -119,6 +119,12 @@ def _real_extract(self, url):
             'upload_date': publish_time,
             'creator': singer,
             'description': lrc_content,
+            'subtitles': {
+                'zh-CN': [{
```

Comment:
new code is added to filter out non-lrc plain text lyrics.
And, doing language detect would require a language model, which requires a lot of code. But introducing new dependencies doesn't sounds good. any advice?